pkg/phlaredb/head.go

```diff
@@ -1003,6 +1004,7 @@ func (h *Head) flush(ctx context.Context) error {
 	h.meta.Files = files
 	h.meta.Stats.NumProfiles = uint64(h.profiles.index.totalProfiles.Load())
 	h.meta.Stats.NumSamples = h.totalSamples.Load()
+	h.meta.Compaction.Sources = []ulid.ULID{h.meta.ULID}
```

Comment:
When prometheus does this it also sets compaction level to `1`

Reply:
https://github.com/prometheus/prometheus/blob/76dd9b5470e6f49330a6420c5d6133bea40a71ea/tsdb/compact.go#L499C1-L500